compose.yml

```diff
@@ -96,3 +96,4 @@ volumes:
 networks:
   cdp-tenant:
     driver: bridge
+    name: cdp-tenant
```

Comment:
Adding named driver means multiple independent compose specs can share common network.

Helpful for frontend and backend using their own compose specs without tight coupling here.  Especially across those services containing more than one service.